solutions/77.js

```diff
@@ -18,4 +18,22 @@ const solution = (string) =>{
   return data;
 };
 
-module.exports = {solution};
+const solution1 = (string) =>{
+  let i = 0;
+  let j = 0;
```

Comment:
This is a working solution. Good start!  Now try to implement this solution without using extra variables `j` and `k`.  Hint: initialize `obj` before entering the while loop.